### PR TITLE
CGMES. Clean up logs related to regulating controls

### PR DESCRIPTION
**Pull request by zamarrenolm**
**Please check if the PR fulfills these requirements**

- [X] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

Minor improvement

**What is the current behavior?**

Log warning messages are reported for equipment without associated regulating control objects.

**What is the new behavior (if this is a feature change)?**

Generators, shunt compensators, static var compensators and tap changers may not have an associated regulating control. We do not need to report this as "missing" data in input CGMES files.
Log messages have been kept, but the log level has been reduced to the lowest one (`TRACE`) and completed with the identifier of the equipment.

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForShuntCompensators.java [hunk 1]
```diff
@@ -75,7 +77,8 @@ private void apply(ShuntCompensator shuntCompensator, CgmesRegulatingControlForS
         // But no regulating control information has been found
         // We create default regulation data
         if (rcId == null) {
-            context.missing("Regulating Control ID not defined");
+            // No need to report as missing, just log a message
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForShuntCompensators.java:80]: Can you remove both comment "// No need to report as missing, just log a message". If you are sure about the solution, you don't need a comment.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForGenerators.java [hunk 2]
```diff
@@ -70,7 +72,8 @@ private void apply(Generator gen, CgmesRegulatingControlForGenerator rc) {
 
         String controlId = rc.regulatingControlId;
         if (controlId == null) {
-            context.missing("Regulating control Id not defined");
+            // No need to report as missing, just log a message
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForGenerators.java:75]: To be removed too.

Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForGenerators.java]: removed

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java [hunk 3]
```diff
@@ -72,7 +74,8 @@ private void apply(StaticVarCompensator svc, CgmesRegulatingControlForStaticVarC
 
         String controlId = rc.regulatingControlId;
         if (controlId == null) {
-            context.missing("Regulating control Id not defined");
+            // No need to report as missing, just log a message
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java:77]: Here too.

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java [hunk 4]
```diff
@@ -327,7 +329,9 @@ private RegulatingControl getTapChangerControl(CgmesRegulatingControl rc) {
 
         String controlId = rc.regulatingControlId;
         if (controlId == null) {
-            context.missing("Regulating control ID not defined");
+            // A tap changer may not have an associated tap changer control,
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java:332]: Here too.

**Review by annetill — CHANGES_REQUESTED** [review]

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java (hunk 3 above)
Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForStaticVarCompensators.java]: comment removed

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java (hunk 4 above)
Comment (zamarrenolm) [reply cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForTransformers.java]: comment removed

File: cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EquivalentInjectionConversion.java [hunk 5]
```diff
@@ -126,7 +128,8 @@ private Regulation getRegulation() {
         boolean regulationCapability = p.asBoolean("regulationCapability", false);
         regulation.status = p.asBoolean("regulationStatus", false) && regulationCapability;
         if (!p.containsKey("regulationStatus") || !p.containsKey(REGULATION_TARGET)) {
-            context.missing(String.format("Missing regulationStatus or regulationTarget for EquivalentInjection %s. Voltage regulation is considered as off.", id));
+            // No need to report as missing, just log a message
```
Comment (annetill) [inline cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EquivalentInjectionConversion.java:131]: Here too :-)

**Review by annetill — APPROVED** [review]

**Pull request merged**